Samples/Client/SwiftClient/SwiftClient/VideoStreamViewController.swift

```diff
@@ -0,0 +1,640 @@
+//
+//  VideoStreamViewController.swift
+//  SwiftClient
+//
+//  Created by Michael Perel on 9/24/17.
+//  Copyright © 2017 Michael Perel. All rights reserved.
+//
+
+import UIKit
+import WebRTC
+
+class VideoStreamViewController: UIViewController {
+    
+    @IBOutlet weak var pickerView: UIPickerView!
+    var otherPeers = [Int: String]()
+    var myId: Int!
+    var heartBeatTimer: Timer!
+    var heartBeatTimerIsRunning = false
+    let heartbeatIntervalInSecs = 5
+    let server = Config.server
+    let localName = "ios_client"
+    var messageCounter = 0
+    var peerConnection: RTCPeerConnection!
+    var peerConnectionFactory = RTCPeerConnectionFactory()
+    var newIceCandidatePeerId = -1
+    var peerListDisplay = ["Select"]
+    var hangingGet: URLSessionTask!
+    var request: URLSessionDataTask!
+    @IBOutlet weak var renderView: RTCEAGLVideoView!
+    var videoTrack: RTCVideoTrack!
+    var videoStream: RTCMediaStream!
+    var inputChannel: RTCDataChannel!
+    lazy var navTransform: [CGFloat] = {
+        [unowned self] in
+        self.matCreate()
+        }()
+    var navHeading: CGFloat = 0.0
+    var navPitch: CGFloat = 0.0
+    var navLocation: [CGFloat] = [ 0.0, 0.0, 0.0 ]
+    var isFingerDown = false
+    var fingerDownX: CGFloat = 0
+    var fingerDownY: CGFloat = 0
+    var downPitch: CGFloat = 0.0
+    var downHeading: CGFloat = 0.0
+    var downLocation: [CGFloat] = [ 0.0, 0.0, 0.0 ]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        RTCInitializeSSL()
```

Comment:
is it possible for this view to be loaded multiple times without being `deinit`-ed? if so we should not call this hook multiple times

Reply:
Although the view is only initialized once, I think best practice would be to move it to AppDelegate in case the app changes in the future.

Reply:
makes sense, i agree